esp/scm/ws_dali.ecm

```diff
@@ -143,6 +143,73 @@ ESPrequest [nil_remove] GetClientsRequest
 {
 };
 
+ESPStruct [nil_remove] DaliStatEntry
+{
+    string Name;
+    string Value;
+    string Type;
```

Comment:
Does this have value?  It can be derived from the name, and the current logic to populate it is wrong.